appium/webdriver/extensions/screen_record.py

```diff
@@ -12,16 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Any, Union
+from typing import TYPE_CHECKING, Any, TypeVar, Union
 
 from selenium import webdriver
 
 from ..mobilecommand import MobileCommand as Command
 
+if TYPE_CHECKING:
+    # noinspection PyUnresolvedReferences
```

Comment:
[Question] Is this for Pycharm?

Reply:
exactly. Pycharm has its own linter, which shows warnings